--- rte-0.5.1/src/Makefile.am.orig	2002-12-25 09:48:28.000000000 +0100
+++ rte-0.5.1/src/Makefile.am	2003-11-11 00:29:46.419039600 +0100
@@ -11,10 +11,6 @@
 if BACKEND_MP1E
 mp1e = ../mp1e
 mp1e_libs = \
-	$(mp1e)/video/libvideo.la \
-	$(mp1e)/audio/libaudio.la \
-	$(mp1e)/systems/libsystems.la \
-	$(mp1e)/common/libcommon.la \
 	$(mp1e)/libmp1e.la
 endif
 
--- rte-0.5.1/mp1e/Makefile.am.orig	2002-12-14 01:43:43.000000000 +0100
+++ rte-0.5.1/mp1e/Makefile.am	2003-11-11 00:31:56.305293888 +0100
@@ -6,7 +6,6 @@
 
 if BACKEND_MP1E
 
-SUBDIRS = video audio systems common
 DIST_SUBDIRS = video audio systems common
 EXTRA_DIST =
 
@@ -19,6 +18,7 @@
 
 ## XXX remove options.h
 libmp1e_la_SOURCES = b_mp1e.c b_mp1e.h options.h
+libmp1e_la_LIBADD = -lmp1e_common
 
 else
 
--- rte-0.5.1/test/Makefile.am.orig	2002-10-25 14:22:03.000000000 +0200
+++ rte-0.5.1/test/Makefile.am	2003-11-11 00:54:08.936703120 +0100
@@ -10,7 +10,7 @@
 	-D_GNU_SOURCE -D_REENTRANT -D_LARGEFILE64_SOURCE \
 	-Wall -Wunused -Wmissing-prototypes -Wmissing-declarations
 
-LDADD = $(top_builddir)/src/.libs/librte.a $(PTHREAD_LIB) -lm
+LDADD = $(top_builddir)/src/librte.la $(PTHREAD_LIB) -lm
 
 ## rte_test_INCLUDES = $(ESD_CFLAGS)
 ## rte_test_LDADD = $(ESD_LIBS) $(PTHREAD_LIB) -lm ../src/.libs/librte.a
